### PR TITLE
Xcode 8 and Swift 2.3 support

### PR DESCRIPTION
**Pull request by ishkawa**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8
```
Comment (kylef) [inline .travis.yml]: Do these changes break compatibility with Xcode 7 (current stable)?

**Comment by kylef** [comment]
Thanks for the pull request @ishkawa.

File: .travis.yml (hunk 1 above)
Comment (ishkawa) [reply .travis.yml]: Currently, this PR breaks compatibility with Xcode 7 by the diff below.
https://github.com/kylef/WebLinking.swift/pull/6/files#diff-6f2529f6d403a440ce762df1db755907R102

We can support both versions of Xcode if we add `#if swift(>=2.3)` there.
I'll work on it.

**Comment by ishkawa** [comment]
I've fixed compatibility with Xcode 7 and set up Travis CI jobs both for Xcode 7 and 8.
https://travis-ci.org/kylef/WebLinking.swift/builds/157338264

**Comment by kylef** [comment]
Awesome. Thanks for the work here @ishkawa

**Comment by maxsz** [comment]
@kylef thanks for merging this! It would be awesome, if you could also draft a new release with this. Thanks 🍻

**Pull request merged**